.github/workflows/dependabot-auto-approve-always.yml

```diff
@@ -0,0 +1,29 @@
+name: Dependabot auto-approve all updates
+on: pull_request
+
+permissions:
+  pull-requests: write
+
+jobs:
+  dependabot:
+    runs-on: ubuntu-latest
+    if: ${{ github.actor == 'dependabot[bot]' }}
+    strategy:
+      matrix:
+        dependencyStartsWith:
+          - '@checkernetwork/prettier-config'
+          - neostandard
+          - prettier
+          - typescript
```

Comment:
I would only auto approve minor changes, why do you think breaking changes are fine as well?

Reply:
PRs to update depending on the outcome of this discussion:
- https://github.com/CheckerNetwork/piece-indexer/pull/178
- https://github.com/CheckerNetwork/node/pull/726

Reply:
I'm ok with auto approving listing PRs but not formatting PRs. I would consider a breaking change in formatting something I'd like to review

Reply:
I see your point. I prefer to keep this simple and have the same setup for both neostandard and eslint, so I'll rework my PR to only auto approve minor changes.

Reply:
See 8976475

Reply:
@juliangruber LGTY now?